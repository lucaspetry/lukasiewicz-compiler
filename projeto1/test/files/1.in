int a = 9

int b
a = 10
