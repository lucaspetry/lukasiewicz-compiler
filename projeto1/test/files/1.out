int var: a = 9
int var: b
= a 10
